pkg/solver/doc.go

```diff
@@ -0,0 +1,5 @@
+// Package solver contains the implementation of the "workspace routing solver" which provides che-specific
```

Comment:
if this is only for "workspace routing solver", shouldn't it be in it's name, rather than generic `solver`? When this operator will have more responsibilities and potentially more "solvers", it will be hard to find.
If it won't be hard to rename later when we know better what we need, I'm ok keeping it as is for now.